website/settings/defaults.py

```diff
@@ -349,9 +349,11 @@ def parent_dir(path):
 ECSARXIV_CROSSREF_USERNAME = None
 ECSARXIV_CROSSREF_PASSWORD = None
 
-# ror
-OSF_ROR_ID = '05d5mza29'
-OSF_GRID_ID = 'grid.466501.0'
```

Comment:
I see the different path that `ROR_ID` takes to get serialized, but was `GRID_ID` dropped?